web-test-runner.config.js

```diff
@@ -12,9 +12,9 @@ const globalCss = sass.compile('./src/components/core/styles/global.scss', { loa
 
 const browsers = isCIEnvironment
   ? [
-      playwrightLauncher({ product: 'chromium' }),
-      playwrightLauncher({ product: 'firefox' }),
-      playwrightLauncher({ product: 'webkit' }),
+      playwrightLauncher({ product: 'chromium', concurrency: 1 }),
```

Comment:
Can you write a comment why we need the concurrency?

Reply:
I'm not really sure why it has problems but I'll write a comment saying that it's necessary to stay like that